airflow-core/src/airflow/models/hitl.py

```diff
@@ -0,0 +1,59 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import sqlalchemy_jsonfield
+from sqlalchemy import Boolean, Column, String, Text
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.ext.hybrid import hybrid_property
+
+from airflow.models.base import Base
+from airflow.settings import json
+from airflow.utils.sqlalchemy import UtcDateTime
+
+
+class HITLResponseModel(Base):
```

Comment:
I feel `HITLResponse(Base)` without the model is enough since we already know this to be an sqlalchemy model. I also wonder if we should just have it as `HITL(Base)` why the response?

Reply:
I'll change it to `HITLDetail` for now because of the time zone difference, but I'm open to other name suggestions since this one isn't ideal either. I think `HITLTask` could be confusing, as it doesn't represent a task, and we already have the concept of a task in Airflow. On its own, `HITL` doesn’t make much sense to me, as it doesn’t refer to a  🤦‍♂️

Reply:
but let's keep the naming thing discussed in https://github.com/apache/airflow/pull/52868#discussion_r2193883283